docs/getting_started/dashboard/dashboard_block_reference.md

```diff
@@ -0,0 +1,170 @@
+
+## Orders by status block
+
+Displays a chart presenting orders split by status, with their number and percentage.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+- **Statuses** - Set the statuses of orders that should be included in the list: Pending, Processing, Cancelled, Completed. Default value = All.
+- **Periods** - Set the time period: All time, Last day, Last week, Last month. Default value = Last month.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of orders and submit your changes.
+
+## Products by category block
+
+Displays a chart presenting the products split by category, together with the percentage.
+Some products can be assigned to multiple categories, or be uncategorized.
+You can choose to show or hide uncategorized products in the chart.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+- **Categories limit** - Set a maximum number of categories to be displayed.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
+
+## Products with lowest stock block
+
+Displays a table of products with the lowest stock.
+Products are sorted based on Stock value (sorted from lowest to highest stock).
+
+The table contains the following columns: Name, Image, Code, Category, Type, Variant, Stock.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+- **Number of products to be displayed** - Set a number of products to be displayed. Default value = 10.
+- **Stock** - Enter the number of products in stock. Default value = 10.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
+
+## Quick actions block
+
+Displays most popular/used actions and shortcuts, for example, **Create content**.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+- **Actions** - Select actions to be displayed as shortucts: Create content, Create form, Create product, Create catalog, Create company.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of quick actions and submit your changes.
+
+## Recent activity block
+
+Displays a list of recent activity of all users.
+It also includes a link to view all activities available in Admin tab.
+
+Recent activity block contains the following data: action time, User reference (avatar, first and last name) with a link to the user profile (if available),
+Activity type with the context.
+
+Available filters are: Users, Activity type, Activity area (default value = All), Number of activities (default value = 5, maximum value = 10).
```

Comment:
This is a part of block properties. But some labels are still not updated. Check: https://issues.ibexa.co/browse/IBX-7486

Reply:
For now, all the labels in the block properties, are (and have to be) consistent with what is visible on the UI. 
They will be updated when all the related PRs will be merged.